src/build/build.ts

```diff
@@ -59,10 +56,13 @@ export async function build(
     js: Object.assign({minify: true}, options.js),
   };
 
-  logger.info(`Preparing build...`);
-  await del([unbundledBuildDirectory, bundledBuildDirectory]);
+  if (options.insertDependencyLinks) {
+    logger.debug(
+        `Additional dependency links will be inserted into application`);
+  }
 
-  logger.info(`Building application...`);
+  logger.info(`Deleting build/ directory...`);
+  await del([buildDirectory]);
```

Comment:
And I've been burned by auto-cleaning, which has removed files I needed, especially when combined with other bugs.